debian/control

```diff
@@ -50,6 +50,7 @@ Build-Depends: bc,
                python (>= 2.7),
                python-all-dev,
                python-nose,
+	       python-prettytable,
```

Comment:
agreed. we debated about "make check" dependencies machinery before, iirc. but no acceptable result back then. i should learn to live with it.